addons/stock_account/migrations/13.0.1.1/post-migration.py

```diff
@@ -33,6 +33,7 @@ def _prepare_common_svl_vals(move, product):
         "remaining_value": 0.0,
         "quantity": 0.0,
         "old_product_price_history_id": None,
+        "account_move_id": move["account_move_id"],
```

Comment:
Is this always available or it will give `KeyError` if no real time accounting?

Reply:
I think so, the `def get_stock_moves(` method always return the account move line, even if it is null.

Reply:
And being `NULL`, I suppose that this is translated to `None` by psycopg2. Isn't there any problem with that?

Reply:
I migrated an actual customer with this, and for sure there were stock moves in/out the company without journal entries created. And it did not fail. In any case we can do like this:
```suggestion
        "account_move_id": move_account_move_id,
```
where:
`move_account_move_id = move.get("account_move_id", False)`

That would be bad for performance?

Reply:
I'm just asking as I don't know it, so no problem at all. Anyways, your code is not impacting performance, except populating the extra variable, that you can put inline:

`"account_move_id": move.get("account_move_id", False),`